src/components/Dropdown/Dropdown.module.css

```diff
@@ -63,3 +62,24 @@
 .sbui-dropdown__trigger {
   @apply border-none bg-transparent p-0 focus:ring-0;
 }
+
+.sbui-dropdown__arrow {
+  @apply fill-current text-bg-primary-light dark:text-bg-secondary-dark;
+  @apply border border-lightmode dark:border-darkmode;
```

Comment:
Border classes, i think, should be `border` and `dark:border-dark`

Reply:
Right it's pulling the default, even for dark mode